spec/models/authenticator/ldap_spec.rb

```diff
@@ -163,6 +164,20 @@ def authenticate
     let(:username) { 'alice' }
     let(:password) { 'secret' }
 
+    context "when using LDAP" do
+      let(:config) { super().merge(:ldap_role => true) }
```

Comment:
@jrafanie I uncovered the reason. 

super without arguments will invoke the superclass method with the original params. To pass no params the empty parens are needed.